Source/WebKit/WebProcess/WebPage/WebFrame.h

```diff
@@ -288,6 +291,7 @@ class WebFrame : public API::ObjectImpl<API::Object::Type::BundleFrame>, public
     std::optional<TransactionID> m_firstLayerTreeTransactionIDAfterDidCommitLoad;
 #endif
     std::optional<NavigatingToAppBoundDomain> m_isNavigatingToAppBoundDomain;
+    std::optional<SafeBrowsingCheckOngoing> m_isSafeBrowsingCheckOngoing;
```

Comment:
I think we should replace all use of optional<SafeBrowsingCheckOngoing> with just SafeBrowsingCheckOngoing::No